NS 7.9 release notes (#536)
NethServer/docs: administrator-manual/en/release_notes.rst

.. only:: nsent

    - ISO release 7.9.2009 "final" replaces any previous ISO

    - This release is based on `CentOS 7 (2009) <https://wiki.centos.org/Manuals/ReleaseNotes/CentOS7>`_

    - CentOS 7 will receive security updates until 2024-06-30

Major changes on 2020-11-XX
---------------------------

* ISO release 7.9.2009 "final" replaces any previous ISO 7.8.2003

* New installations of Nextcloud honor the StartTLS setting of the Active Directory accounts provider.
  As old installations ignore that setting and always send clear-text passwords, it is recommended

>   to upgrade an existing installation to the new behavior, if its remote AD accounts provider

Reviewer: ```suggestion
  to upgrade an existing installation to the new behavior. If its remote AD account provider
```

Author: I was thinking the same...

What do you think to remove completely "If remote AD account provider supports StartTLS"? It should be true if sssd is configured for it.

Reviewer: I do not have too much faith on Microsoft AD.
What about rephrasing to:
```
Make sure the remote AD supports StartTLS, then run the following command ::
```

Then add also a line like "NethServer AD always support StartTLS".

Author: About the latest one: https://docs.nethserver.org/en/v7/accounts.html#bind-to-a-remote-ldap-server

That section must be adjusted for the new Cockpit UI. We could add the note about StartTLS there and a link the section here.

Reviewer: I agree

>   supports StartTLS. Run the following commands ::

Reviewer: ```suggestion
  supports StartTLS, run the following commands ::
```

.. only:: nsent

    - ISO release 7.9.2009 "final" replaces any previous ISO

    - This release is based on `CentOS 7 (2009) <https://wiki.centos.org/Manuals/ReleaseNotes/CentOS7>`_

    - CentOS 7 will receive security updates until 2024-06-30

Major changes on 2020-11-XX
---------------------------

* ISO release 7.9.2009 "final" replaces any previous ISO 7.8.2003

* The old Server Manager (namely Nethgui) is not available by default on new installations.
  To configure the system access the new Server Manager on port ``9090``.

  Old Server Manager can be still installed from :guilabel:`Software Center`.

> * Since |product| 7.8.1908 CGP (Collectd Graph Panel), EveBox and Rspamd UI were available only on HTTPS port 980,

Reviewer: ```suggestion
* Until |product| 7.8.1908, CGP (Collectd Graph Panel), EveBox and Rspamd UI were available only on HTTPS port 980,
```

  under the old Server Manager Apache instance named ``httpd-admin``.
  These applications are now available also on port 443 and will be accessible even if the old Server Manager has
  not been installed.

  Access to the above applications can be restricted: see the respective manual pages for :ref:`CGP <cgp_restict_access-section>`,

>   :ref:`Rspamd <rspamd-web-interface-section>` and :ref:`Evebox <evebox_restrict_access-section>`.

Reviewer: ```suggestion
  :ref:`Rspamd <rspamd-web-interface-section>` and :ref:`EveBox <evebox_restrict_access-section>`.
```

* ISO release 7.9.2009 "final" replaces any previous ISO 7.8.2003

> * Sice NethServer 7.8.1908 CGP (Collectd Graph Panel), EveBox and Rspamd UI where available only on HTTPS port 980,

Author: ```suggestion
* Since |product| 7.8.1908 CGP (Collectd Graph Panel), EveBox and Rspamd UI were available only on HTTPS port 980,
```

* ISO release 7.9.2009 "final" replaces any previous ISO 7.8.2003

> * As default, users belonging to the ``wheel`` group can now access the systems using SSH.

Author: - Explain what ``wheel`` is
- "As default" could be interpreted as "affecting existing systems" 

```suggestion
* In newly installed systems, users belonging to the ``wheel`` group (i.e. those created during the ISO installation procedure) can log in via SSH.
```

>   This behavior can be changed using the ``AllowLocalGroups`` property.

Author: ```suggestion
  This behavior can be changed with the ``openssh/AllowLocalGroups`` property.
```

  It accepts a comma separated list of group names in ``/etc/groups`` that are granted full SSH and SFTP access.

>   To restore previous behavior, run the following commands: ::

Author: I don't want to pass the idea that the new behavior is applied to existing systems.

```suggestion
  If SSH access for the `wheel` administrative group is not desired, run the following commands: ::
```

  :guilabel:`System > Users & Groups > [Account provider] > Edit provider`.
  See also :ref:`dedicated-service-account`.

>   To prevent errors during the upgrade to Nextcloud, the ``mail`` and ``theming`` have been disabled.

Author: Let's split the Nextcloud in two points: new/existing 

```suggestion
* To prevent errors during the upgrade of existing Nextcloud installations, the ``mail`` and ``theming`` applications have been disabled.
```

  After each upgrade, both applications should be manually updated and re-enabled by accessing
  Nextcloud administration interface.

* Netdata is now installed by default to serve charts for the Server Manager.
  Some plugins have been disabled to reduce resource usage.

>   To re-enable such plugins see `netdata configuration <https://docs.nethserver.org/projects/nethserver-devel/en/latest/nethserver-netdata.html>`_ .

Author: What about saying

> To enable those plugins see...

Reviewer: Done
